src/main/java/seedu/weme/model/ModelManager.java

```diff
@@ -51,13 +54,39 @@ public ModelManager(ReadOnlyWeme weme, ReadOnlyUserPrefs userPrefs) {
         versionedWeme = new VersionedWeme(weme);
         this.userPrefs = new UserPrefs(userPrefs);
         filteredMemes = new FilteredList<>(versionedWeme.getMemeList());
+        filteredStagedMemeList = new FilteredList<>(versionedWeme.getStagedMemeList());
+        filteredImportMemeList = new FilteredList<>(versionedWeme.getImportList());
         filteredTemplates = new FilteredList<>(versionedWeme.getTemplateList());
     }
 
     public ModelManager() {
         this(new Weme(), new UserPrefs());
     }
 
+    //=========== Export/Import ==============================================================================
+
+    @Override
+    public List<Path> getExportPathList() {
+        return versionedWeme.getExportPathList();
+    }
+
+    @Override
+    public void importMeme() throws IOException {
```

Comment:
```suggestion
    public void importMemes() throws IOException {
```
Fix the rest as well